contrib/cirrus/runner.sh

```diff
@@ -2,13 +2,89 @@
 
 set -eo pipefail
 
-# This script is intended to be called by automation or humans,
-# from a specially configured environment.  Depending on the contents
-# of various variable, entirely different operations will be performed.
+# This script runs in the Cirrus CI environment, invoked from .cirrus.yml
```

Comment:
Also (since I'm working on it) this and most of the scripts can be invoked by hand, assuming your inside a `hack/get_ci_vm.sh` environment.  Maybe say, "...from `.cirrus.yml` or a specially constructed environment"?

Reply:
Fixed, but with a TBI.